src/rpc/server.cpp

```diff
@@ -169,11 +170,39 @@ UniValue stop(const JSONRPCRequest& jsonRequest)
             }.ToString());
     // Event loop will exit after current HTTP requests have been handled, so
     // this reply will get back to the client.
-    StartShutdown();
-    if (jsonRequest.params[0].isNum()) {
-        MilliSleep(jsonRequest.params[0].get_int());
+    if(rpcallowstop)
```

Comment:
Could you follow [Coding Style (C++)](https://github.com/bitcoin/bitcoin/blob/master/doc/developer-notes.md#coding-style-c)?

Reply:
Sure thing

Reply:
Done

Reply:
```
$ git diff -U0 HEAD~2.. src/rpc/server.cpp | ./contrib/devtools/clang-format-diff.py -p1 -v
--- src/rpc/server.cpp	(before formatting)
+++ src/rpc/server.cpp	(after formatting)
@@ -170,36 +170,28 @@
             }.ToString());
     // Event loop will exit after current HTTP requests have been handled, so
     // this reply will get back to the client.
-    if (rpcallowstop)
-    {
+    if (rpcallowstop) {
         StartShutdown();
         if (jsonRequest.params[0].isNum()) {
             MilliSleep(jsonRequest.params[0].get_int());
         }
         return "Bitcoin server stopping";
-    }
-    else
-    {
+    } else {
         // Check if IP is 127.0.0.1
         std::string ip;
-        for (int i = 0; i < jsonRequest.peerAddr.length(); i++)
-        {
-            if (jsonRequest.peerAddr[i] == ':')
-            {
+        for (int i = 0; i < jsonRequest.peerAddr.length(); i++) {
+            if (jsonRequest.peerAddr[i] == ':') {
                 break;
             }
             ip += jsonRequest.peerAddr[i];
         }
-        if (ip == "127.0.0.1")
-        {
+        if (ip == "127.0.0.1") {
             StartShutdown();
             if (jsonRequest.params[0].isNum()) {
                 MilliSleep(jsonRequest.params[0].get_int());
             }
             return "Bitcoin server stopping";
-        }
-        else
-        {
+        } else {
             throw JSONRPCError(RPC_FORBIDDEN_BY_SAFE_MODE, "You are not allowed to stop");
         }
     }
```

Reply:
> You can use the provided [clang-format-diff script](https://github.com/bitcoin/bitcoin/blob/master/contrib/devtools/README.md#clang-format-diffpy) tool to clean up patches automatically before submission.